src/js/puzzles.js

```diff
@@ -333,28 +333,28 @@ var puzzlesArray = [
   id: 48,
   sentence: 'Are you sure this will be sun?',
   word: 'sun',
-  hint: 'Time of my life',
+  hint: 'time of my life',
```

Comment:
While we're changing this, I'd prefer the possessive pronouns to be indeterminate (i.e., time of one's life).

Reply:
Fixed on my latest commit.